packages/web-react/src/components/Breadcrumbs/demo/BreadcrumbsCustom.tsx

```diff
@@ -35,14 +36,22 @@ const BreadcrumbsCustom = () => {
         };
 
         return (
-          <>
-            {index !== 0 && <Icon name="chevron-right" />}
-            <li key={`BreadcrumbsItem_${item.title}`}>
+          <Fragment key={itemKey}>
```

Comment:
Can we unify the usage of the fragment? React.Fragment vs Fragment

Reply:
Yes. I have changed this to `<Fragment>` only.